mantis-server/mantis-server-worker-client/src/test/java/io/mantisrx/server/worker/client/SseWorkerConnectionTest.java

```diff
@@ -91,4 +105,274 @@ public void testStreamContentDrops() throws Exception {
+            }
+
+            @Override
+            public ChannelFuture connect(SocketAddress remoteAddress, ChannelPromise promise) {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture connect(SocketAddress remoteAddress, SocketAddress localAddress, ChannelPromise promise) {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture disconnect(ChannelPromise promise) {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture close(ChannelPromise promise) {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture deregister(ChannelPromise promise) {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture write(Object msg) {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture write(Object msg, ChannelPromise promise) {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture writeAndFlush(Object msg, ChannelPromise promise) {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture writeAndFlush(Object msg) {
+                return null;
+            }
+
+            @Override
+            public ChannelPromise newPromise() {
+                return null;
+            }
+
+            @Override
+            public ChannelProgressivePromise newProgressivePromise() {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture newSucceededFuture() {
+                return null;
+            }
+
+            @Override
+            public ChannelFuture newFailedFuture(Throwable cause) {
+                return null;
+            }
+
+            @Override
+            public ChannelPromise voidPromise() {
+                return null;
+            }
+
+            @Override
+            public <T> Attribute<T> attr(AttributeKey<T> key) {
+                return null;
+            }
+
+            @Override
+            public <T> boolean hasAttr(AttributeKey<T> key) {
+                return false;
+            }
+
+            @Override
+            public int compareTo(Channel o) {
+                return 0;
+            }
+        };
+
+        client.trackConnection(dummyChannel);
+        logger.info("Connection tracker size: {}", client.connectionTrackerSize());
+        assertEquals(1, client.connectionTrackerSize());
+
+        client.closeConn();
+
+        logger.info("Connection tracker size: {}", client.connectionTrackerSize());
+        assertEquals(0, client.connectionTrackerSize());
+        logger.info("Channel active: {}, channel open: {}, channel writable: {}", dummyChannel.isActive(), dummyChannel.isOpen(), dummyChannel.isWritable());
+        assertFalse(dummyChannel.isActive());
```

Comment:
These assertions are pointless no? They are hardcoded to return false. We can just remove.

Reply:
I don't fully understand what you mean. 
I am closing the connection via the `closeConn` method and then testing if the original channel does get closed. If it does not, then the assertion will fail.

Reply:
Does dummyChannel.isActive() always return false? I don't see where it could be set to true.

Reply:
That is a great point. Removed those assertions.